YDotNet/Document/Types/Branches/Branch.cs

```diff
@@ -12,17 +13,18 @@ namespace YDotNet.Document.Types.Branches;
 /// </summary>
 public abstract class Branch : UnmanagedResource
 {
-    private readonly EventSubscriber<EventBranch[]> onDeep;
+    private readonly EventSubscriberFromId<EventBranch[]> onDeep;
 
-    internal protected Branch(nint handle, Doc doc, bool isDeleted)
+    protected internal Branch(nint handle, Doc doc, bool isDeleted)
         : base(handle, isDeleted)
     {
         Doc = doc;
 
-#pragma warning disable CA1806 // Do not ignore method results
-        onDeep = new EventSubscriber<EventBranch[]>(
+        BranchId = isDeleted ? null : BranchChannel.Id(handle);
```

Comment:
Perhaps we can just rely on default here and avoid the nullabilty?

Reply:
@SebastianStehle Is [this](https://github.com/y-crdt/ydotnet/commit/9d88a194d9dc7d8231baff7d9b379bba84fa6309) what you mean?

Reply:
Yes